server/index.js

```diff
@@ -59,5 +60,7 @@ socket.init(server);
 
 const PORT = 8080;
 
+runAgenda();
```

Comment:
what if agenda function throws an error? does it crash the app? 
don't we need any error handler here or inside `runAgenda` function even with  `catch(noOp)`??